backend/app/requests/census.py

```diff
@@ -28,18 +28,23 @@ def census_info_by_service_area_request(app, flask_request, engine):
     try:
         request_json = flask_request.get_json(force=True)
         service_area_ids = request_json['service_area_ids']
+        service_area_ids.sort()
     except (json.JSONDecodeError, KeyError):
         raise InvalidFormat(message='Invalid JSON format.')
 
+    dataset_hint = request_json.get('dataset_hint', None)
```

Comment:
Same comment as on the other PR about including `dataset_hint` in non-adequacy endpoints.

Reply:
Removed in other PR